src/main/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherTaxServiceImpl.java

```diff
@@ -80,4 +86,48 @@ protected boolean validateNRATaxInformation(DisbursementVoucherDocument document
         }
         return true;
     }
+    
+	public boolean isForeignVendorWithTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId) {
+
+		return isForeignVendor(payeeTypeCode, vendorHeaderId) && !paymentReasonDoesNotRequireTaxReviewForForeignVendor(paymentReasonCode);
+	}
+	
+	public boolean isForeignVendoAndNOTaxReviewRequired(String payeeTypeCode,
+			String paymentReasonCode, Integer vendorHeaderId) {
+
+		return isForeignVendor(payeeTypeCode, vendorHeaderId) && paymentReasonDoesNotRequireTaxReviewForForeignVendor(paymentReasonCode);
+	}
+	
+	protected boolean isForeignVendor(String payeeTypeCode, Integer vendorHeaderId){
+		if (payeeTypeCode.equals(KFSConstants.PaymentPayeeTypes.VENDOR) && vendorService.isVendorForeign(vendorHeaderId)){ 
+			return true;
+		}
+		
+		return false;
+	}
+	
+	protected boolean paymentReasonDoesNotRequireTaxReviewForForeignVendor(String paymentReasonCode){
+		if (parameterEvaluatorService.getParameterEvaluator(DisbursementVoucherDocument.class, CuDisbursementVoucherConstants.PAYMENT_REASONS_THAT_DO_NOT_REQUIRE_TAX_REVIEW_FOR_FOREIGN_VENDOR, paymentReasonCode).evaluationSucceeds()) {
```

Comment:
Parameter evaluators are capable of treating a parameter's values as a whitelist or blacklist depending on the parameter's Allow/Deny flag. There are other places in KFS that rely on this flag for treating the values as a blacklist. Your code should work as-is, but if you want to make things consistent with similar setup in KFS, you could change this around so that evaluationSucceeds() means tax review IS required, and then set the parameter's flag to Deny so that its elements will be treated as a blacklist. (This may also require a parameter name change for consistency.) Let me know if this makes sense in the event that you do intend to go forward with the change.